Capstone/News/NewsViewController.m

```diff
@@ -0,0 +1,63 @@
+//
+//  NewsViewController.m
+//  Capstone
+//
+//  Created by Abel Kelbessa on 7/7/22.
+//
+
+#import "NewsViewController.h"
+#import "UIImageView+AFNetworking.h"
+#import "NewsCell.h"
+
+@interface NewsViewController ()<UITableViewDelegate,UITableViewDataSource>
+@property(strong, nonatomic) NSMutableArray<NSDictionary *> *collegeNews;
+@property(strong, nonatomic) NSMutableArray *college1;
+@property (weak, nonatomic) IBOutlet UITableView *tableView;
+
+@end
+
+@implementation NewsViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    self.tableView.dataSource = self;
+    self.tableView.delegate = self;
+    self.college1 = [[NSMutableArray alloc] init];
+    [self fetchData];
+}
+
+- (void)fetchData {
```

Comment:
😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭

can we move the data fetching to the API manager? i think it was ok the first time (hence the earlier TODO comment), but now that it's in 3 places in the code, it's time to refactor